README.md

```diff
@@ -62,55 +58,98 @@ As you can see, most caching libraries only take care of (1) and (4) (well, and
 
 Just setting REDIS_URL in your environment is enough.
 
-REDIS_URL=redis://redis:6379/2
+```sh
+export REDIS_URL=redis://redis:6379/2
+```
```

Comment:
this is actually bad guidance - people should almost never put lock and cache in the same redis database

just delete it and stick with the separate LOCK_URL and CACHE_URL below